integrated_channels/degreed2/client.py

```diff
@@ -303,6 +335,36 @@ def update_content_metadata(self, serialized_data):
             patch_url,
             course_id
         )
+        # once course is updated successfully, we need to do 2 more steps
+        # 1. Fetch skills from enterprise-catalog
+        client = EnterpriseCatalogApiClient()
+        metadata = client.get_content_metadata_content_identifier(
+            enterprise_uuid=self.enterprise_configuration.enterprise_customer.uuid,
+            content_id=external_id
+        )
+        LOGGER.info(
+            generate_formatted_log(
+                self.enterprise_configuration.channel_code(),
+                self.enterprise_configuration.enterprise_customer.uuid,
+                None,
+                None,
+                f'[Degreed2Client] metadata: {metadata}'
+            )
+        )
+        # 2. Transmit to degreed
+        skills = metadata['skill_names']
+        if skills:
+            self.assign_course_skills(external_id, skills)
+            LOGGER.info(
+                generate_formatted_log(
+                    self.enterprise_configuration.channel_code(),
+                    self.enterprise_configuration.enterprise_customer.uuid,
+                    None,
+                    None,
+                    f'[Degreed2Client] transmitted skills: {metadata["skill_names"]},'
+                    f'for course: {course_id}'
+                )
+            )
```

Comment:
Following the DRY principe, this whole block can be put inside a helper method and be called inside create and update course method.